### PR TITLE
Msi z790p support

### PR DESCRIPTION
**Pull request by miczyg1**

### CODE REVIEW

File: src/lib/coreboot_table.c [hunk 1]
```diff
@@ -403,6 +405,45 @@ void __weak lb_board(struct lb_header *header) { /* NOOP */ }
  */
 void __weak lb_spi_flash(struct lb_header *header) { /* NOOP */ }
 
+/*
```
Comment (mkopec) [inline src/lib/coreboot_table.c:408]: Does this work? I was under the impression this code won't work on newer revisons of coreboot. That's why this was rewritten as I mentioned here: https://github.com/Dasharo/coreboot/pull/352

**Review by mkopec — CHANGES_REQUESTED** [review]

File: src/lib/coreboot_table.c (hunk 1 above)
Comment (miczyg1) [reply src/lib/coreboot_table.c:408]: Yup it still works, but I will apply your change here.

Comment (miczyg1) [reply src/lib/coreboot_table.c:408]: @mkopec fixed by cherry-picking https://github.com/Dasharo/coreboot/pull/355/commits/4d866ff8ffabe30624eaf380872ee2850b82ac2c

**Review by mkopec — APPROVED** [review]
Thanks

**Comment by miczyg1** [comment]
@mkopec added small fixes after noticing review on gerrit:
https://github.com/Dasharo/coreboot/pull/355/commits/e66c9c3cc30c49f4fd90907041649e7f8b5eadc6
https://github.com/Dasharo/coreboot/pull/355/commits/32752defff7fb57925a55173738dacee2fbd639f

**Pull request merged**